src/ublox/ubx_cfg.h

```diff
@@ -17,25 +17,31 @@
 //#define UBLOX_PARSE_SVINFO
 //#define UBLOX_PARSE_CFGNAV5
 //#define UBLOX_PARSE_MONVER
+//#define UBLOX_PARSE_HNR_PVT
 
 //--------------------------------------------------------------------
 // Identify the last UBX message in an update interval.
 //    (There are two parts to a UBX message, the class and the ID.)
 // For coherency, you must determine which UBX message is last!
 // This section *should* pick the correct last UBX message for NEO-6M devices.
 
-#define UBX_LAST_MSG_CLASS_IN_INTERVAL ublox::UBX_NAV
-
 #if defined(UBLOX_PARSE_DOP)
```

Comment:
Yes you're right, that's a typo from when I copied from my own project into a clean branch.